utils/meminfo.go

```diff
@@ -0,0 +1,308 @@
+/*
```

Comment:
Please write some unit tests for the function in this new package. `golang.org` has documentation on writing unit tests and where to put them.

Reply:
I actually took this package code from here: https://github.com/guillermo/go.procmeminfo, which comes with tests. So the question is, do you want me to take their tests too, or we go get this package during make? The reason I took his code was because what if the author deletes the repo in future? So I felt safe to make a copy of it.

Reply:
Yeah, don't duplicate the code. Just import the package. I forked it into https://github.com/nci/go.procmeminfo and you can import it from there. No need to write additional tests if that package has a testsuite. Thanks!

Reply:
okay i now made importing from github.com/nci/procmeminfo. In addition, i think it would be a good strategy to make forks of tidy pakcages into nci. I wouldn't worry about big packages with a lot of contributors but tiny packages can be a concern if their repos are lost. For example, bitbucket.org/monkeyforecaster/geometry. I'll create an issue for it.